common/lib/xmodule/xmodule/modulestore/tests/test_semantics.py

```diff
@@ -0,0 +1,119 @@
+from xmodule.modulestore.tests.utils import PureModulestoreTestCase, VersioningModulestoreBuilder, MongoModulestoreBuilder, SPLIT_MODULESTORE_SETUP
+from xmodule.modulestore.exceptions import ItemNotFoundError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.factories import CourseFactory
+
+class AboutItemSemantics(PureModulestoreTestCase):
+    """
+    Verify the behavior of About items (which are intentionally orphaned, DIRECT_ONLY/autopublished)
+    blocks intended to store snippets of course content.
+    """
+
+    __test__ = False
+
+    def setUp(self):
+        super(AboutItemSemantics, self).setUp()
+        self.course = CourseFactory.create(
+            org='test_org',
+            number='999',
+            run='test_run',
+            display_name='My Test Course',
+            modulestore=self.store
```

Comment:
It's specifying this attribute: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/factories.py#L80-L86